greenweb/settings/testing.py

```diff
@@ -22,3 +22,4 @@
         # "dramatiq.middleware.Retries",
     ],
 }
+TRELLO_REGISTRATION_EMAIL_TO_BOARD_ADDRESS = "mail-to-board@trello.com"
```

Comment:
Question: for testing environment, should we use an email address that is not external? E.g. `mail-to-board@localhost`. I'm not sure if the current setup will successfully prevent sending external emails